code/modules/admin/battle_royale.dm

```diff
@@ -116,8 +116,8 @@ GLOBAL_DATUM(battle_royale, /datum/battle_royale_controller)
 /client/proc/battle_royale()
 	set name = "Battle Royale"
 	set category = "Fun"
```

Comment:
Can admins actually see the `Fun` tab if they don't have `+FUN`? If not, the category needs to be changed or they'll have an entire `Fun` table with nothing but this one verb they can't use until roundend on it.

Reply:
We can, there's three or four verbs in there for mods at the moment.

Reply:
Never mind, we don't have the fun tab, moving it to Adminbus tab.

Reply:
Fixed